web/components/answers/answers-panel.tsx

```diff
@@ -101,6 +105,20 @@ export function AnswersPanel(props: {
   const userBetsByAnswer = groupBy(userBets, (bet) => bet.answerId)
   const unfilledBets = useUnfilledBets(contract.id)
 
+  const [shouldShowLimitOrderChart, setShouldShowLimitOrderChart] = useState(
+    safeLocalStorage?.getItem(SHOW_LIMIT_ORDER_CHARTS_KEY) === 'true' ?? false
+  )
+
+  const handleToggleLimitOrderCharts = () => {
+    const newValue = !shouldShowLimitOrderChart
+
+    setShouldShowLimitOrderChart(newValue)
+    safeLocalStorage?.setItem(
+      SHOW_LIMIT_ORDER_CHARTS_KEY,
+      newValue ? 'true' : 'false'
+    )
+  }
+
```

Comment:
This looks fine, but could you try using `usePersistentLocalState`:

Example:

```
const [lover, setLover] = usePersistentLocalState<
    Row<'lovers'> | undefined | null
  >(undefined, `lover-${user?.id}`)
  ```

Reply:
Right on, I figured I was missing a hook that did that. Updated!